src/plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -111,6 +111,8 @@ const handleCourierRequest = async ({
     return aggs.toDsl(metricsAtAllLevels);
   });
 
+  requestSearchSource.setFilterPath(['*', '-**.key_as_string']);
```

Comment:
nit: Could we include a comment here so that its purpose is clear? Perhaps a link to the key_as_string docs or some indication that this is specific to date histogram responses.

Reply:
Doesn't changing this affect most visualizations, and not only `date histogram`?
If so, maybe it's worth updating the title of the issue?